Helper/Data.php

```diff
@@ -635,8 +635,10 @@ public function getCorrectVCL($pageCacheConfig, $major_release)
     {
         if ($major_release == "4") {
             return $pageCacheConfig->getVclFile(\Magento\PageCache\Model\Config::VARNISH_4_CONFIGURATION_PATH);
-        } else {
+        } elseif ($major_release == "5") {
             return $pageCacheConfig->getVclFile(\Magento\PageCache\Model\Config::VARNISH_5_CONFIGURATION_PATH);
+        } else {
+            return $pageCacheConfig->getVclFile(\Magento\PageCache\Model\Config::VARNISH_6_CONFIGURATION_PATH);
```

Comment:
Have we confirmed that `Config::VARNISH_6_CONFIGURATION_PATH` is defined in older Magento Framework versions that we declare to support below (e.g. v100.0)?

Reply:
No, I haven't confirmed version backward compatibility and have no practical way of doing so. 

I snowflake the upgraded server; PHP version, Magento version and Magento extension each time. 

The section.magento Vagrant box is broken afaik and so its not easy to doing version compatibility checks.

Reply:
It looks like that `VARNISH_6_CONFIGURATION_PATH` constant was [introduced in Magento v2.3.3](https://github.com/magento/magento2/commit/21768739df8d8d68ae5db36eaccc8e54c3401044). Do we know if we need to support customers with Magento older than that?

Reply:
> It looks like that `VARNISH_6_CONFIGURATION_PATH` constant was [introduced in Magento v2.3.3](https://github.com/magento/magento2/commit/21768739df8d8d68ae5db36eaccc8e54c3401044). Do we know if we need to support customers with Magento older than that?

Probably, however, why wouldn't they use a previous version of the Magento extension?

Reply:
>Probably, however, why wouldn't they use a previous version of the Magento extension?

I'm not familiar with the Magento Extension Store. If it has multiple versions of each extensions available concurrently to choose from and each version of the extension makes it clear which version of Magento it will work with, this seems reasonable.